Resources/Prototypes/Entities/Clothing/Head/misc.yml

```diff
@@ -219,7 +219,7 @@
   - type: AddAccentClothing
     accent: OwOAccent
   - type: StaticPrice
-    price: 15000
+    price: 500
```

Comment:
If we're removing these from the maints lockers, feel free to bump them up to 5000 (to be in line with the 3x cat ears you get from the bundle)